### PR TITLE
Update jetbrains extension - fix crashing search recent projects command

### PR DESCRIPTION
**Pull request by dgrzedzielski**
## Description

Last fix for extension unfortunately didn't fix it fully.
It fixes error

```
TypeError: Cannot read properties of undefined (reading '0')

b8:util.ts:235:44

---
232:   if (tool.toolName === undefined) {
233:     return [];
234:   }
235:   const build = channel.history?.toolBuilds[0] ?? {};
236:   const directoryPatterns = build?.tool?.intelliJProperties?.directoryPatterns ?? [];
237:   const recentProjectsFilenames =
238:     channel.history?.toolBuilds[0]?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
---

recent.js:52:1759
    at async Promise.all (index 0)
async cl:recent.js:52:1262
```

Mentioned in this issue: https://github.com/raycast/extensions/issues/15887

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/jetbrains/src/util.ts [hunk 1]
```diff
@@ -232,10 +232,10 @@ const globFromChannel = async (tool: Tool, channel: ChannelDetail) => {
   if (tool.toolName === undefined) {
     return [];
   }
-  const build = channel.history?.toolBuilds[0] ?? {};
+  const build = channel.history?.toolBuilds?.[0] ?? {};
   const directoryPatterns = build?.tool?.intelliJProperties?.directoryPatterns ?? [];
   const recentProjectsFilenames =
-    channel.history?.toolBuilds[0]?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
+    channel.history?.toolBuilds?.[0]?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
```
Comment (gdsmith) [inline extensions/jetbrains/src/util.ts:238]:
```suggestion
    build?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
```

**Review by gdsmith — APPROVED** [review]

File: extensions/jetbrains/src/util.ts [hunk 2]
```diff
@@ -232,10 +232,10 @@ const globFromChannel = async (tool: Tool, channel: ChannelDetail) => {
   if (tool.toolName === undefined) {
     return [];
   }
-  const build = channel.history?.toolBuilds[0] ?? {};
+  const build = channel.history?.toolBuilds?.[0] ?? {};
   const directoryPatterns = build?.tool?.intelliJProperties?.directoryPatterns ?? [];
   const recentProjectsFilenames =
-    channel.history?.toolBuilds[0]?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
+    build?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
```
Comment (gdsmith) [inline extensions/jetbrains/src/util.ts:238]: line is too short to split now 🥹

Comment (dgrzedzielski) [reply extensions/jetbrains/src/util.ts]: fixed, I thought that commiting through GH may be not the best idea 😄

**Review by gdsmith — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by bigplayer-ai** [comment]
thanks @gdsmith and @dgrzedzielski this solution worked.

**Pull request merged**